warp/Network/Wai/Handler/Warp/HTTP2/Receiver.hs

```diff
@@ -232,9 +232,12 @@ control FrameGoAway _ _ Context{controlQ} = do
 
 control FrameWindowUpdate header bs Context{connectionWindow} = do
     WindowUpdateFrame n <- guardIt $ decodeWindowUpdateFrame header bs
-    !w <- (n +) <$> atomically (readTVar connectionWindow)
+    !w <- atomically $ do
```

Comment:
Ack. I've added a strictness annotation from within the STM (cf. a48c7aa.) In that particular instance since w1 was returned and immediately evaluated by the bang pattern I guess it was fine.

I can rebase on master and squash if you prefer a clean history. Let me know.
Thanks for the review!